lib/rsc/storyblok-component.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { forwardRef } from "react";
+import {
+  getComponent,
+  getEnableFallbackComponent,
+  getCustomFallbackComponent,
+} from "./index";
+import type { SbBlokData } from "../types";
+
+interface StoryblokComponentProps {
+  blok: SbBlokData;
+  [key: string]: unknown;
```

Comment:
Same reasoning as before, I suggest narrow the object type

Reply:
see above

Reply:
https://github.com/storyblok/storyblok-react/pull/1201#discussion_r1796571378